lib/senders/email.js

```diff
@@ -36,6 +36,7 @@ module.exports = function (log) {
     'unblockCode': 'new-unblock',
     'verifyEmail': 'welcome',
     'verifyLoginEmail': 'new-signin',
+    'verifyLoginCodeEmail': 'new-signin-code',
```

Comment:
Careful with naming here, a "signin code" is an existing and unrelated concept in this codebase so this might be confusing to future us.  Maybe "new-signin-verify-code" or similar would make it clearer?